src/pages/RealtimeMapPage.tsx

```diff
@@ -73,14 +72,19 @@ export default function RealtimeMapPage() {
   }
 
   //TODO (another PR and another issue) load from url like in another pages.
-  const [from, setFrom] = useState('2023-05-01T12:00:00+02:00') // arbitrary default value. this date is not important
-  const [to, setTo] = useState('2023-05-01T12:01:00+02:00')
+  const fiveMinutesAgo = moment().subtract(5, 'minutes')
+  const now = moment()
```

Comment:
define these constants outside the component, on top.
we don't want a new date to be created every render